addons/account/views/account_move_views.xml

```diff
@@ -1342,7 +1358,7 @@
             <field name="name">Bills</field>
             <field name="res_model">account.move</field>
             <field name="view_mode">tree,kanban,form</field>
-            <field name="view_id" ref="view_invoice_tree"/>
+            <field name="view_id" ref="view_in_invoice_tree"/>
```

Comment:
same in `action_move_in_refund_type` & `action_move_in_receipt_type`

Reply:
@smetl That wasn't done for out invoices; you sure about that?